packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/react/DevMenuAwareReactNativeHost.kt

```diff
@@ -0,0 +1,17 @@
+package expo.modules.devmenu.react
+
+import android.app.Application
+import android.os.Bundle
+import com.facebook.react.ReactInstanceManager
+import com.facebook.react.ReactNativeHost
+import expo.modules.devmenu.protocoles.DevMenuDelegateProtocol
+
+/**
+ * Basic [ReactNativeHost] which knows about expo-dev-menu and implements [DevMenuDelegateProtocol].
+ */
+abstract class DevMenuAwareReactNativeHost(application: Application)
+  : ReactNativeHost(application), DevMenuDelegateProtocol {
+  override fun appInfo(): Bundle? = null
```

Comment:
```suggestion
abstract class DevMenuAwareReactNativeHost(application: Application)
  : ReactNativeHost(application),
    DevMenuDelegateProtocol
{
  override fun appInfo(): Bundle? = null
```
Yeah, that would be my preference regarding multiline class signature 😉